nonprofit-board-management.php

```diff
@@ -946,6 +951,39 @@ private function validate_gravatar( $user_email ) {
 
       return $has_valid_avatar;
     }
+
+    /**
+     * Check if the Block Editor is being used.
+     * 
+     * @return bool True if the Block Editor is being used, false if not.
+     */
+    private function is_block_editor_in_use() {
+
+      $use_block_editor = get_option( 'classic-editor-replace' ) === 'no-replace';
+
+      // Check if the Gutenberg plugin is installed and activated.
+      $gutenberg_plugin = ! ( false === has_filter( 'replace_editor', 'gutenberg_init' ) );
```

Comment:
Do you think we still need this code since 5.0 is launching tomorrow? I assume they're getting rid of the plugin since Gutenberg will be in core.

Reply:
I could see it going either way - I don't want to make any assumptions. Who knows how long people will wait before upgrading but could be using the Gutenberg plugin in that time frame.

That being said if it helps keep this simple I'll go ahead and remove it.

Reply:
This check has been removed.